backend/src/resolvers/user_management.spec.js

```diff
@@ -309,3 +310,58 @@ describe('change password', () => {
     })
   })
 })
+
+describe('do not expose private RSA key', () => {
+  let client
+
+  const queryUser = () => {
+    return gql`
+      query($queriedUserSlug: String) {
+        User(slug: $queriedUserSlug) {
+          id
+          privateKey
+        }
+      }`
+  }
+
+  const action = async () => {
+    // Generate user with "privateKey" via 'CreateUser' mutation instead of using the factories "factory.create('User', {...})", see above.
+    const variables = {
+      id: 'bcb2d923-f3af-479e-9f00-61b12e864667',
+      password: 'xYz',
+      slug: 'apfel-strudel',
+      name: 'Apfel Strudel',
+      email: 'apfel-strudel@test.org'
+    }
+    await client.request(gql`
+      mutation($id: ID, $password: String!, $slug: String, $name: String, $email: String) {
+        CreateUser(id: $id, password: $password, slug: $slug, name: $name, email: $email) {
+          id
+        }
+      }`, variables
+    )
+  }
+
```

Comment:
```suggestion
describe('unauthenticated query of "publicKey"', () => {
  it('returns publicKey', () => {
    // check that an RSA key pair has been created at all
  })
})
```

Reply:
I have done an authenticated query for „publicKey“, because I am not sure for the future, if unauthenticated queries of user data should be possible.

Would be nice to get a review by @mattwr18 or @ulfgebhardt , because @roschaefer can not review his own PRs.